server/base/src/main/java/org/apache/accumulo/server/manager/state/TabletManagementScanner.java

```diff
@@ -70,7 +70,9 @@ public class TabletManagementScanner implements ClosableIterator<TabletManagemen
         final ClientTabletCache locator = ClientTabletCache.getInstance(context, tid);
         final Set<String> locations = new HashSet<>();
         final List<Range> failures = locator.findTablets(context, ALL_TABLETS_RANGE, (ct, r) -> {
-          locations.add(ct.getTserverLocation().orElse("UNASSIGNED"));
+          if (ct.getTserverLocation().isPresent()) {
+            locations.add(ct.getTserverLocation().orElseThrow());
```

Comment:
```suggestion
            locations.add(ct.getTserverLocation().orElseThrow(()-> new IllegalStateException("tablet locator tserver location is missing"));
```

Reply:
Since the code is calling isPresent on the previous line I don't think we should ever see an exception. I was wondering if there was a better way to do this and found Optional has an ifPresnet(Consumer) method where the consumer is only called when its present.  I used that method and it really simplified this code an avoided the call to orElseThrow.  Made that change in 1c8e238482529b258aa353f55043a2c029cdc897